src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -43,32 +43,32 @@
       <_NET50TargetingPackVersion>5.0.0</_NET50TargetingPackVersion>
       <_WindowsDesktop50RuntimePackVersion>5.0.3</_WindowsDesktop50RuntimePackVersion>
       <_WindowsDesktop50TargetingPackVersion>5.0.0</_WindowsDesktop50TargetingPackVersion>
-      <_AspNet50RuntimePackVersion>5.0.4</_AspNet50RuntimePackVersion>
+      <_AspNet50RuntimePackVersion>5.0.5</_AspNet50RuntimePackVersion>
       <_AspNet50TargetingPackVersion>5.0.0</_AspNet50TargetingPackVersion>
 
       <_NETCoreApp31RuntimePackVersion>3.1.13</_NETCoreApp31RuntimePackVersion>
       <_NETCoreApp31TargetingPackVersion>3.1.0</_NETCoreApp31TargetingPackVersion>
       <_WindowsDesktop31RuntimePackVersion>3.1.12</_WindowsDesktop31RuntimePackVersion>
       <_WindowsDesktop31TargetingPackVersion>3.1.0</_WindowsDesktop31TargetingPackVersion>
-      <_AspNet31RuntimePackVersion>3.1.13</_AspNet31RuntimePackVersion>
+      <_AspNet31RuntimePackVersion>3.1.14</_AspNet31RuntimePackVersion>
       <_AspNet31TargetingPackVersion>3.1.10</_AspNet31TargetingPackVersion>
 
       <_NETCoreApp30RuntimePackVersion>3.0.3</_NETCoreApp30RuntimePackVersion>
       <_NETCoreApp30TargetingPackVersion>3.0.0</_NETCoreApp30TargetingPackVersion>
-      
+
       <_NETCoreApp31RuntimePackVersion>3.1.13</_NETCoreApp31RuntimePackVersion>
       <_NETCoreApp31TargetingPackVersion>3.1.0</_NETCoreApp31TargetingPackVersion>
 
       <_WindowsDesktop30RuntimePackVersion>3.0.3</_WindowsDesktop30RuntimePackVersion>
       <_WindowsDesktop30TargetingPackVersion>3.0.0</_WindowsDesktop30TargetingPackVersion>
-      
+
       <_WindowsDesktop31RuntimePackVersion>3.1.13</_WindowsDesktop31RuntimePackVersion>
       <_WindowsDesktop31TargetingPackVersion>3.1.0</_WindowsDesktop31TargetingPackVersion>
 
       <_AspNet30RuntimePackVersion>3.0.3</_AspNet30RuntimePackVersion>
       <_AspNet30TargetingPackVersion>3.0.1</_AspNet30TargetingPackVersion>
-      
-      <_AspNet31RuntimePackVersion>3.1.13</_AspNet31RuntimePackVersion>
+
+      <_AspNet31RuntimePackVersion>3.1.14</_AspNet31RuntimePackVersion>
```

Comment:
I think this duplication is a bug. Specifically, it looks like pieces of 5.0.x branches were merged over when it wasn't necessary. Specifically, these two commits look strange: 

https://github.com/dotnet/installer/commit/273a80d66ebf75c8d295566f33a73c8d254d925d#diff-8c78b9ed6c8d5a033ec06056db1d34b86190a6e188bce3eeb6981c5b1f2a7c5c
https://github.com/dotnet/installer/commit/c01306370ff62d0365251b83dfec44293404ff57#diff-8c78b9ed6c8d5a033ec06056db1d34b86190a6e188bce3eeb6981c5b1f2a7c5c

Reply:
> I think this duplication is a bug. Specifically, it looks like pieces of 5.0.x branches were merged over when it wasn't necessary.

@dsplaisted @wli3❔

Reply:
Yes, this looks like a bad merge undoing some automatic versioning that I think @marcpopMSFT did:

https://github.com/dotnet/installer/commit/c01306370ff62d0365251b83dfec44293404ff57#diff-8c78b9ed6c8d5a033ec06056db1d34b86190a6e188bce3eeb6981c5b1f2a7c5c

Reply:
Unfortunately, we can't use the automatic versioning in previews/main (at least I haven't figured out a number to queue off of) because they don't have stable version numbers to go off of.  The automatic versioning should only be through 5.0.2xx and it looks like a bad merge because it escaped out to 3xx and beyond.

Reply:
I plan on leaving the merge fixes to the repo owners so I'll proceed with this PR as is. Do you want me to file an issue for this so we don't forget? Who should I assign/tag?